Resources/doc/getting_started/the_list_view.rst

```diff
@@ -0,0 +1,215 @@
+The List View
+=============
+
+You've given the admin a nice interface to create and edit blog posts and
+categories. But there is not much to change if the admin doesn't have a list of
+all available blog posts. Because of this, this chapter will teach you more
+about the list view.
+
+If you have created some posts and go to
+http://localhost:8000/admin/app/blogpost/list, you'll see an empty list page.
```

Comment:
I always leave it out of the backticks, so people can actually click on the link when following the guide. Happy to change it though